src/pages/Demo.tsx

```
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Text, View } from 'react-native'
+
+type Props = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)